lib/grape_doc.rb

```diff
@@ -11,11 +12,52 @@
 module GrapeDoc
   def self.generate_doc
     opts = Trollop::options do
-      opt :path, "Resource path", 
-          :type => :string, 
-          :default => File.expand_path(Dir.pwd + "/config/environment") 
+      opt :batch, "Generate documentation for multiple APIs",
+          :short => :B
+      opt :bundle, "Run bundle on child APIs",
+          :short => :b
+      opt :output_dir, "Directory to save documentation file(s)",
+          :type => :string,
+          :default => File.expand_path(Dir.pwd + "/grape_doc")
+      opt :resource_files, "API resource files, relative to project root",
+          :type => :strings,
+          :short => :F,
+          :default => ["./config/environment.rb"]
+      opt :root_api, "Top level API class name",
+          :type => :string
+      opt :stdout, "Print documentation stdout instead of generating files",
+          :short => :s,
+          :default => false
+    end
+
+    FileUtils.mkdir_p(opts[:output_dir]) if !opts[:stdout]
```

Comment:
This is a good use case for unless. It'll make the code a tad bit more readable.

If you don't like unless, I also prefer
```ruby
 FileUtils.mkdir_p(opts[:output_dir]) if opts[:stdout].nil?
```